js/web/lib/wasm/jsep/webgpu/ops/common.ts

```diff
@@ -772,14 +772,14 @@ class ShaderHelperImpl implements ShaderHelper {
     const is1DimensionDispatch = this.normalizedDispatchGroup[1] === 1 && this.normalizedDispatchGroup[2] === 1;
     const paramList = is1DimensionDispatch ? `@builtin(global_invocation_id) global_id : vec3<u32>,
     @builtin(local_invocation_id) local_id : vec3<u32>` :
-                                             `@builtin(local_invocation_index) local_index : u32,
+                                             `@builtin(local_invocation_index) local_idx : u32,
     @builtin(workgroup_id) workgroup_id : vec3<u32>,
     @builtin(num_workgroups) num_workgroups : vec3<u32>`;
```

Comment:
nit: If we can unify the paramList, the code will be more consistent. For example, always add blow builtins no matter it's is1DimensionDispatch or not. But I don't know whether it will bring perf impact. I suppose it will be very small. One advantage is that all ops can unify to use `helper.mainStart()` instead of writing their own in case some builtins are missing. 
```
const paramList = `
@builtin(global_invocation_id) global_id : vec3<u32>,
@builtin(local_invocation_id) local_id : vec3<u32>,
@builtin(local_invocation_index) local_idx : u32,
@builtin(workgroup_id) workgroup_id : vec3<u32>,
@builtin(num_workgroups) num_workgroups : vec3<u32>
`;
const globalIdxDefinition = is1DimensionDispatch ?
        'let global_idx = global_id.x;' :
        `let global_idx = (workgroup_id.z * ... ... `

Reply:
This change required modifying many files.